src/FakeItEasy/Core/FakeManager.EventRule.cs

```diff
@@ -145,8 +138,8 @@ private void RaiseEvent(EventCall call, IEventRaiserArgumentProvider argumentPro
 
             private class EventCall
             {
-                private static readonly Func<EventInfo, MethodInfo> GetAddMethod = e => e.GetAddMethod(true);
-                private static readonly Func<EventInfo, MethodInfo> GetRemoveMethod = e => e.GetRemoveMethod(true);
+                private static readonly Func<EventInfo, MethodInfo> GetAddMethod = e => e.GetAddMethod(true)!;
+                private static readonly Func<EventInfo, MethodInfo> GetRemoveMethod = e => e.GetRemoveMethod(true)!;

```

Comment:
No idea when GetAddMethod or GetRemoveMethod can return null... I don't think it's possible to declare an event without add or remove in C#, but maybe directly in IL? Anyway, I don't know what we should do in such a case...

Reply:
I think this is fine.